hack/install-kwok.sh

```diff
@@ -72,6 +75,9 @@ do
   - op: replace
     path: /spec/template/spec/containers/0/args/2
     value: --manage-nodes-with-label-selector=kwok-partition=${alphabet[i]}
+  - op: replace
+    path: /spec/template/spec/containers/0/args/9
+    value: --node-lease-duration-seconds=0
```

Comment:
I think we should extend this time instead of turning it off, and we need to change the heartbeat check time of the kube-controller-manager (--node-monitor-period, --node-monitor-grace-period), otherwise the node status will move to unknown in about 2 min.